lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -896,7 +885,7 @@ static function ($boolean) : ?int {
      */
     public function convertFromBoolean($item) : ?bool
     {
-        if (in_array($item, $this->booleanLiterals['false'], true)) {
+        if ((self::BOOLEAN_LITERALS[$item] ?? null) === false) {
```

Comment:
This expression must remain as clear as it used to be: `if isset(/* ... */) return`;